scripts/bootstrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# For local development we need this script tp create collections in a local jamdb.
+# For deployment this will need to be run manaually.
+
+source ~/.bashrc
+
+workon `lsvirtualenv -b |grep jam`
```

Comment:
Side notes for future improvements (as much for myself as anyone else): this script contains a lot of assumptions. It fails vigorously under the following conditions:

- For people who don't use `virtualenvwrapper` and/or name their virtual environments in a very specific way. 
- Fails if the collection already exists
- Fails if there is no `~/.bashrc` file
- It (possibly) does not add permissions for the OSF user by default. (when logging in as OSF, I now get a 403 when hitting the URL http://localhost:1212/v1/id/namespaces/experimenter from the experimenter app)